contents/feedback/progress/quest-progress/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { CircleCheckBigIcon } from "@yamada-ui/lucide"
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  Center,
+  Container,
+  HStack,
+  Heading,
+  Progress,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+const mockDatas = [
+  {
+    content: "Solve 5 question",
+    current: 5,
+    max: 5,
+  },
+  {
+    content: "Solve 10 question",
+    current: 10,
+    max: 10,
+  },
+  {
+    content: "Solve 15 question",
+    current: 12,
+    max: 15,
+  },
+]
+
+const QuestProgress: FC = () => {
+  return (
+    <Container m="auto">
+      <Heading>Daily Quests</Heading>
+      <HStack justifyContent="end">
+        <Text>Time remaining：12hours</Text>
+      </HStack>
```

Comment:
Please use half-width colon and add a space.

```suggestion
      <HStack justifyContent="end">
        <Text>Time remaining: 12 hours</Text>
      </HStack>
```

Reply:
OK